services/data/data/service.py

```diff
@@ -148,7 +169,7 @@ def get_product_detail(
             return ServiceException(500, self.get_user_id(user), str(exp)).to_dict()
 
     @rpc
-    def refresh_cache(self, user: Dict[str, Any] = None, use_cache: bool = False) -> dict:
```

Comment:
The refresh_cache method can be accessed from the REST interface in the gateway. This endpoint if protected by auth -> a user object will be passed -> the `user` parameter is required

Reply:
That's true, although the user object is not needed in that function. I'll put it back for now, but we need a smarter implementation of auth that passes the user object only if needed and otherwise checks the user's permissions/roles